src/store/index.js

```diff
@@ -16,12 +16,19 @@ import user_interface from './modules/user_interface'
 import uiSync from './modules/uiSync'
 
 import UiSyncPlugin from './plugins/ui_sync'
+import createPersistedState from 'vuex-persistedstate'
 
 Vue.use(Vuex)
 
+// Todo: research more into other state variables could benefit from a persisted state and seperate into a module
```

Comment:
Perhaps add a bit to this comment saying that this is used to persist the OWM report state across page refreshes - i.e. _why_ we're doing this